src/test/scala/edu/ie3/simona/config/ConfigFailFastSpec.scala

```diff
@@ -596,86 +596,6 @@ class ConfigFailFastSpec extends UnitSpec with ConfigTestData {
         val checkBaseRuntimeConfigs =
           PrivateMethod[Unit](Symbol("checkBaseRuntimeConfigs"))
 
-        val defaultString: String = "default"
-
-        "throw an InvalidConfigParameterException if the list of UUIDs of the base model config is empty" in {
-          val baseRuntimeConfig = ConfigFactory.parseString(
-            """simona.runtime.participant.load = {
-              |  defaultConfig = {
-              |    calculateMissingReactivePowerWithModel = false
-              |    uuids = []
-              |    scaling = 1.3
-              |    modelBehaviour = "profile"
-              |    reference = "power"
-              |  }
-              |  individualConfigs = []
-              |}""".stripMargin
-          )
-          val config =
-            baseRuntimeConfig.withFallback(typesafeConfig).resolve()
-          val simonaConfig = SimonaConfig(config)
-
-          intercept[InvalidConfigParameterException] {
-            ConfigFailFast invokePrivate checkBaseRuntimeConfigs(
-              simonaConfig.simona.runtime.participant.load.defaultConfig,
-              simonaConfig.simona.runtime.participant.load.individualConfigs,
-              defaultString,
-            )
-          }.getMessage shouldBe "There has to be at least one identifier for each participant."
-        }
-
-        "throw an InvalidConfigParameterException if a valid single key is given and the UUID of the base model config is not valid" in {
-          val baseRuntimeConfig = ConfigFactory.parseString(
-            """simona.runtime.participant.load = {
-              |  defaultConfig = {
-              |    calculateMissingReactivePowerWithModel = false
-              |    uuids = ["blabla"]
-              |    scaling = 1.3
-              |    modelBehaviour = "profile"
-              |    reference = "power"
-              |  }
-              |  individualConfigs = []
-              |}""".stripMargin
-          )
-          val config =
-            baseRuntimeConfig.withFallback(typesafeConfig).resolve()
-          val simonaConfig = SimonaConfig(config)
-
-          intercept[InvalidConfigParameterException] {
-            ConfigFailFast invokePrivate checkBaseRuntimeConfigs(
-              simonaConfig.simona.runtime.participant.load.defaultConfig,
-              simonaConfig.simona.runtime.participant.load.individualConfigs,
-              defaultString,
-            )
-          }.getMessage shouldBe "Found invalid UUID 'blabla' it was meant to be the string 'default' or a valid UUID."
-        }
-
-        "throw an InvalidConfigParameterException if the UUID of the base model config is not valid" in {
```

Comment:
This test would be still helpful, wouldn't it?

Reply:
Since we no longer need a `default` string, the elements are no longer checked in `ConfigFailFast`. With this PR only a log statement is made, if the list of uuids is not empty.

Reply:
The way I see it this test was about malformed UUIDs though, right?

Reply:
Yes, but the test only checks the uuid of the base (default) config. I will adapt the test, to test the uuids of the individual configs.